src/blocks/column/edit.js

```diff
@@ -137,11 +133,8 @@ function SectionEdit( props ) {
 		} else {
 			addUniqueID( uniqueID, clientId );
 		}
-		const isInQueryBlock = getInQueryBlock( context, inQueryBlock );
-		if ( attributes.inQueryBlock !== isInQueryBlock ) {
-			attributes.inQueryBlock = isInQueryBlock;
-			setAttributes( { inQueryBlock: isInQueryBlock } );
-		}
+
+		setAttributes( { inQueryBlock: getInQueryBlock( context, inQueryBlock ) } );
```

Comment:
I have a memory of there being some reason I had to set attributes.inQueryBlock even though I was using setAttributes. Can we leave this like it was just to prevent a previous issue?